### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Databases for assessment_Keyboard Trap: Keyboard tab focus is getting trapped inside databases grid and is not coming out of grid.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19843]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG2.1.2; #Keyboard Trap;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Repro Steps:

1. Install Azure data studio app and install Azure SQL migration extension
2. Connect to SQL server, tab till respective server, right click and select manage
3. Tab till Azure SQL Migration in sitemap and press enter key
4. Tab till migrate to azure sql and press enter
5. Navigate to databases grid in "Databases for assessment" in step 1.
6. Verify that keyboard focus is getting trapped inside databases grid and is not coming out of grid.

### Actual Result:
Keyboard tab focus is getting trapped inside databases grid and is not coming out of grid.

### Expected Result:
Keyboard tab focus should not get trapped inside databases grid and should come out of grid. User should be able to navigate to each row using arrow keys and on clicking space/enter key database should get selected. On pressing tab key, focus should come out of grid.

### Attachment:
![Keyboard trap_](https://user-images.githubusercontent.com/83570638/176095930-feb0c794-5760-458b-bd28-a6834055694a.gif)

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. User is able to navigate to all the databases present in the grid using arrow keys and no keyboard trap is observed. Hence closing the bug.
Env: Azure data studio Version 1.40.0-insider(user setup)
![19843_fixed](https://user-images.githubusercontent.com/83570638/192484874-d54fdf67-f240-447b-a910-dd8380b7d365.gif)

**Comment by VijayaBharathyA** [comment]
#Closed;
